plugins/view-resources/src/components/FixedColumn.svelte

```diff
@@ -54,6 +57,7 @@
 
 <div
   bind:this={element}
+  bind:clientWidth={elWidth}
```

Comment:
We used resizeObserver, since clientWidth was really slow with svelte.

```suggestion
  use:resizeObserver={(element) => {
    elWidth = element.clientWidth
  }}
```